content/en/code_analysis/troubleshooting/_index.md

```diff
@@ -54,6 +76,13 @@ For issues with Datadog Software Composition Analysis, include the following inf
 - The name of the branch you ran the analysis on
 - The list of dependency files in your repository (such as `package-lock.json`, `requirements.txt`, or `pom.xml`)
 
+### Results are not being surfaced in the Datadog UI
+
+If you are running on a non-Github repository, ensure you first commit on your default branch (for example, a branch name like
```

Comment:
"If you are running Code Analysis on a non-GitHub repository, ensure that the first scan is ran on your default branch ...."